src/extension/background-script/connectors/alby.ts

```diff
@@ -254,11 +254,18 @@ export default class Alby implements Connector {
         token: this.config.oAuthToken, // initialize with existing token
       });
 
+      authClient.on("tokenRefreshed", async (token: Token) => {
+        await this._updateOAuthToken(token);
```

Comment:
I do not think async and await are needed here, there's only one method call. Also, js-sdk just uses .emit() and I don't believe it can wait for the result.

Reply:
resolved